test/integration/scheduler/plugin/plugin_test.go

```diff
@@ -562,14 +562,14 @@ func TestPreFilterPlugin(t *testing.T) {
 			preFilterPlugin.failPreFilter = test.fail
 			preFilterPlugin.rejectPreFilter = test.reject
 			// Create a best effort pod.
-			pod, err := createPausePod(testCtx.ClientSet,
-				initPausePod(&pausePodConfig{Name: "test-pod", Namespace: testCtx.NS.Name}))
+			pod, err := testutils.CreatePausePod(testCtx.ClientSet,
+				testutils.InitPausePod(&testutils.PausePodConfig{Name: "test-pod", Namespace: testCtx.NS.Name}))
```

Comment:
Also maybe those function's aren't super controversial, but I would actually move them not to testutils, but rather to test/integration/scheduler/utils

Reply:
+1
I just said the same before I saw your comment :sweat_smile: